lib/datasource/remote/model/invite_model.dart

```diff
@@ -1,24 +1,29 @@
 import 'package:equatable/equatable.dart';
+import 'package:seeds/utils/string_extension.dart';
 
 class InviteModel extends Equatable {
-  final int? inviteId;
```

Comment:
I still do not understand why these serialization models extends from equatable, I have not seen any of these models compare by value in the project.

Reply:
probably not needed

Reply:
I guess it would be useful in case they're part of a state change. But yeah.

Reply:
I'd prefer to have only things I know I'm going to use. 

> in case they're part of a state change

Actually bloc state clases already has its own Equatable to detect new instances (state changes)